tests/run_tests.sh

```diff
@@ -42,21 +42,27 @@ fi
 if [[ "$subset" == "unit" ]]
 then
   path="tests/unit_tests/"
-  coverage_file=".coveragerc_unit"
+  coverage_arg="--coverage-file .coveragerc_unit"
 elif [[ "$subset" == "integration" ]]
 then
   path="tests/integration_tests/"
-  coverage_file=".coveragerc_integration"
+  coverage_arg="--coverage-file .coveragerc_integration"
 else
-  path="tests/"
-  coverage_file=".coveragerc"
+  echo "Please specify unit or integration tests"
+  exit 1
+fi
+
+if [[ -n "${CURATION_TESTS_FILEPATH}" && -s "${CURATION_TESTS_FILEPATH}" ]]; then
+  test_arg="--test-paths-filepath ${CURATION_TESTS_FILEPATH}"
+else
+  test_arg=""
 fi
 
 if [[ -z ${substring} ]]
 then
-  cmd="tests/runner.py --test-path ${path} ${sdk_dir} --coverage-file ${coverage_file}"
+  cmd="tests/runner.py --test-dir ${path}  ${coverage_arg} ${test_arg}"
 else
-  cmd="tests/runner.py --test-path ${path} ${sdk_dir} --test-pattern $substring --coverage-file ${coverage_file}"
+  cmd="tests/runner.py --test-dir ${path} --test-pattern $substring ${coverage_arg} ${test_arg}"
```

Comment:
One thing I'd like to see here is replacing this with a similar setup to what is done in the `curation.sh` script, namely building an array of positional options to pass to the command, rather than building an elongated string.

There are several reasons for this, not the least of which is it's easier to see the logic and you end up with a cleaner / less error prone output.

It would also allow you to more easily print a debug statement akin to `echo "Executing: tests/runner.py ${runner_args[*]}"`

Reply:
I did a quick and dirty update for this, but for more enhancements, we might need a different ticket.